schema.graphql

```diff
@@ -1,11 +1,21 @@
 type Query {
 
+    ranked_search_results(
+        desired_checkin: String
+        desired_checkout: String
+    ): [DatesCheck!]
+
     dates_and_price_check(
         listing_ids: [Int!]!
+        desired_checkin: String
+        desired_checkout: String
     ): [DatesCheck!]
 
 }
 
 type DatesCheck {
     listing_id: Int!
+    is_available:Boolean
+    checkin_date:String
+    checkout_date:String
```

Comment:
Inconsistent spacing after the colon. Also, in your Python DatesCheck class, `is_available` is a never null, meaning that it should be `is_available: Boolean!`.

Reply:
@shanmu9898 the latest commits still did not solve the inconsistent spacing. It should be:
```
    is_available: Boolean
    checkin_date: String
    checkout_date: String
```
based on how the lines above were formatted.